apps/metadata-editor/src/app/dashboard/sidebar/sidebar.component.ts

```diff
@@ -41,4 +45,20 @@ export class SidebarComponent implements OnInit {
       (orgs, me) => orgs.filter((org) => org.name === me?.organisation)
     )
   }
+
+  logOut() {
+    fetch(this.authService.logoutUrl, {
+      method: 'GET',
+    })
+      .then((response) => {
+        if (response.ok) {
+          window.location.href = 'http://localhost:4200'
```

Comment:
In production there won't be a localhost url. I'd suggest to save the host before logging out and using it here instead. WDYT?

Reply:
Yes indeed, didn't think of that though it's a bit obvious :smile: